pennylane/interfaces/batch/jax.py

```diff
@@ -86,6 +87,28 @@ def execute(tapes, device, execute_fn, gradient_fn, gradient_kwargs, _n=1, max_d
     )
 
 
+def _validate_tapes(tapes):

```

Comment:
This might be a good method to implement for all of the different devices and frameworks! Im not sure which ones support which operations!

Reply:
As far as I know, it's only the JAX interface that we have constraints on. :thinking: